### PR TITLE
T1 mise en place plugin

### PR DESCRIPTION
**Pull request by Fred1511**
1er essai :)

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from .sammo import Sammo
```
Comment (pblottiere) [inline __init__.py:1]: Can we add a header to Python files please? Something like:

``` python
# coding: utf8

__contact__ = "info@hytech-imaging.fr"
__copyright__ = "Copyright (c) 2021 Hytech Imaging"
```

File: metadata.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[general]
+name=Sammo
```
Comment (pblottiere) [inline metadata.txt:2]:
```suggestion
name=Sammo-Boat
```

File: metadata.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[general]
+name=Sammo
+description=Tool for megafauna observations from halieutic boats
+about=Tool for megafauna observations
+version=1.0
+qgisMinimumVersion=3.0
+author=Paul Blottière
+email=xxxxx
```
Comment (pblottiere) [inline metadata.txt:8]:
```suggestion
email=info@hytech-imaging.fr
```

File: metadata.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[general]
+name=Sammo
+description=Tool for megafauna observations from halieutic boats
+about=Tool for megafauna observations
+version=1.0
+qgisMinimumVersion=3.0
+author=Paul Blottière
+email=xxxxx
+repository=xxxxx
```
Comment (pblottiere) [inline metadata.txt:9]:
```suggestion
repository=https://github.com/hytechimaging/sammo-boat/
```

**Comment by pblottiere** [comment]
I just have some minor comments but it looks good to me :+1:

File: src/gui/session.py [hunk 5]
```diff
@@ -8,7 +13,7 @@ def __init__(self, iface):
         self.action = None
 
     def initGui(self):
-        self.action = QAction("Sammo", self.iface.mainWindow())
+        self.action = QAction("Session", self.iface.mainWindow())
```
Comment (Fred1511) [inline src/gui/session.py:16]: Little mistake : this label modification should have been in another commit

File: __init__.py (hunk 1 above)
Comment (Fred1511) [reply __init__.py:6]: Done

**Pull request merged**